win/libsass.vcxproj

```diff
@@ -1,17 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="GitVersion;Main"
-    xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-
+<Project ToolsVersion="14.0" DefaultTargets="GitVersion;Main" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <SASSC_VERSION>[NA]</SASSC_VERSION>
     <LIBSASS_VERSION>[NA]</LIBSASS_VERSION>
   </PropertyGroup>
-
   <Target Name="GitVersion">
-
-    <Exec Command="git -C .. describe --abbrev=4 --dirty --always --tags > libsass-version" />
-    <Exec Command="git -C ../sassc describe --abbrev=4 --dirty --always --tags > sassc-version" />
-
+    <Exec Command="git -C .. describe --abbrev=4 --dirty --always --tags &gt; libsass-version" />
```

Comment:
Yep. It actually prefers escaped strings in the params and values.  :)
I verified it as followed, (please let me know if that is the correct way to verify it):

```powershell
C:\users\adeel\Source\Repos\libsass [master +2 ~0 -0 !]> .\win\bin\Debug\sassc.exe -v
sassc: 3.2.5-14-g5c78
libsass: 3.1.0-603-gc380
sass2scss: 1.0.3
```